src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -152,6 +153,13 @@ describe('UserCredentialsUtils', () => {
             assert(profiles.credentialsFile)
             assert(profiles.credentialsFile[profileName])
         })
+
+        it('generated credentials file can be read and written', async () => {
```

Comment:
This test needs to generate a credentials file of its own to verify access flags against. It is probably "borrowing" the file left by the generateCredentialsFile test (above) since is is using the same filename.

Some options:
* add flag validation to the above test instead, after the call to generateCredentialsFile
* use a unique filename in this test, and call generateCredentialsFile